content/en/get-started/_index.md

```diff
@@ -180,7 +180,8 @@ $ docker run --rm -it -p 4566:4566 -p 4510-4559:4510-4559 localstack/localstack
 
 {{< alert title="Notes" >}}
 - This command pulls the current nightly build from the `master` branch (if you don't have the image locally) and **not** the latest supported version.
-  If you want to use a specific version, use the appropriate tag (for example `localstack/localstack:1.0.0`).
+  If you want to use a specific version, use the appropriate tag. For example to install localstack v1.1.0 in docker use -
```

Comment:
Let's use `<version>` instead of providing an arbitrary version.

Reply:
It would also be good to link the release page here to help the users navigate across the releases.